core/widgets.py

```diff
@@ -0,0 +1,44 @@
+from typing import Any, Dict, Optional
+from django.forms import Widget
+
+class TagsEditor(Widget):
+    input_type = 'hidden'
+    template_name = 'core/widget_tag_editor.html'
+
+    @staticmethod
+    def __unpack_tag(tag):
+        return {'name': tag.name, 'color': tag.color}
+
+    def format_value(self, value: Any) -> Optional[str]:
+        if value is not None and not isinstance(value, str):
+            value = [self.__unpack_tag(tag) for tag in value]
+        return value
+    
+    def get_context(self, name: str, value: Any, attrs) -> Dict[str, Any]:
+        from . import models
+
+        most_tags = models.BabyBuddyTag.objects.order_by(
```

Comment:
Why only "most"?

Reply:
Forgot to comment on this. "most" to prevent DDoS attacks in case someone adds thousands of tags. That will make the database operation very expensive, so I cap it at 256. Just like REST-API caps requests on pages at 100 requests. This however is a direct database operation, so I had to do it myself.